### DISCUSSION TITLE
Crash on now playing query

### DISCUSSION THREAD

**Discussion by fifididi** [ppy/osu#16829] [Q&A] [answered]
**Steps to reproduce**:

1. Open the now playing menu (default F6)
2. Search for a non existing song
3. Spam Enter (two presses are enough)

**Note**:

It seems that the game doesn't crash if you give it something around one second in between presses. 

**Proof**:

https://user-images.githubusercontent.com/40372348/153098701-4e03c591-8768-4f98-9ab6-8f24570a02cd.mp4

**Build**:
2022.205.0-lazer

**Comment by peppy** [comment, marked as answer]
For future reports it would be appreciated if you can include log files.

**Reply by fifididi** [reply]
[runtime.log](https://github.com/ppy/osu/files/8028635/runtime.log)

**Reply by peppy** [reply]
Should be fixed via #16830.
